main.py

```diff
@@ -47,22 +47,21 @@ def get_files(files, dl_file_name):
     ''' Download files only if the do not exist '''
     downloaded_file_names = file_to_set(dl_file_name)
     if not files:
-        print 'No files found to download.'
+        print ('No files found to download.')
         return None
     new_files = []
     for file in files:
         if file['name'] in downloaded_file_names:
-            print 'File "{}" is already downloaded.'.format(
-                os.path.basename(file['name']))
+            print ('File "{}" is already downloaded.'.format(os.path.basename(file['name'])))
             continue
         new_files.append(download_file(file, dl_file_name))
     return new_files
 
 
 def setup():
-    print '\n' + str(datetime.now().strftime('%Y-%m-%d %H:%M:%S'))
-    print '=' * 40
-    print "Fetching results related to " + FACULTY + " faculty"
+    print ('\n' + str(datetime.now().strftime('%d-%b-%Y %H:%M:%S')))
```

Comment:
Thanks for fixing this.
`30-Jan-2019 12:17:02'` looks much better than `2019-01-30 12:19:30` :)

Reply:
Thanks for appreciating 😀